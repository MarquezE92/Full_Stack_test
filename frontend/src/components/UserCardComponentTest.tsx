/* import React from 'react';

const UserCardComponentTest = ({name, city='?', country='?', favorite_sport='?'})=>{
    return(
        <div className='card'>
            <h2>{name}</h2>
            <h3>City: {city}</h3>
            <h3>Country: {country}</h3>
            <h3>Favorite sport: {favorite_sport}</h3>
        </div>
    )
}

export default UserCardComponentTest */