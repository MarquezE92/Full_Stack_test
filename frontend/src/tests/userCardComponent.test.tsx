/* import React from 'react';
import TestRenderer from 'react-test-renderer'
import UserCardComponentTest from '../components/UserCardComponentTest';

describe('CardComponent testing', ()=> {
    test('CardComponent renders correctly', () => {
  // Arrange
  const mockData = {
    name: 'Estefanía Márquez',
    city: 'Mar del Plata',
    country: 'Argentina',
    favourite_sport: 'Basketball'
  };

  
  const treeLine = TestRenderer.create(<UserCardComponentTest {...mockData} />).toJSON()
  expect(treeLine).toMatchSnapshot()
});

})
 */