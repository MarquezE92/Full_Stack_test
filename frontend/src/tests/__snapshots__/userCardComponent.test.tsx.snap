// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardComponent testing CardComponent renders correctly 1`] = `
<div
  className="card"
>
  <h2>
    Estefanía Márquez
  </h2>
  <h3>
    City: 
    Mar del Plata
  </h3>
  <h3>
    Country: 
    Argentina
  </h3>
  <h3>
    Favorite sport: 
    ?
  </h3>
</div>
`;
